build/action.yaml

```diff
@@ -19,43 +19,34 @@ inputs:
   npm-token:
     required: false
     description: The Node Package Manager (npm) authentication token. This token is used to authenticate against the NPM registry.
-
+  s3-bucket-name:
+    required: false
+    description: S3 bucket name to cache node_modules to speed-up dependency installation.
+  s3-bucket-region:
+    required: false
+    description: S3 bucket region to cache node_modules to speed-up dependency installation.
 runs:
   using: composite
   steps:
-    - name: Checkout
-      uses: actions/checkout@v4
-      if: inputs.checkout-repo == 'true'
-      with:
-        fetch-depth: 0
-    - name: Setup tools
-      uses: open-turo/action-setup-tools@v2
-    - name: Check for yarn.lock
-      id: check_yarn_lock
-      uses: andstor/file-existence-action@v3
+    - name: Install dependencies
+      id: install_dependencies
+      uses: open-turo/actions-node/install-dependencies@77305b51ac6b16b03da8a6b770c383167924673a
```

Comment:
To release this, we may want to ship the new `install-dependencies` GHA and then create a follow-up PR so we don't point to commit hashes 😄

Reply:
Yeah I was just using this to test in consumers like web-ui

Reply:
Here we go https://github.com/open-turo/actions-node/pull/67, will merge that and then we can merge this